docs/en/observability/logs-troubleshooting.asciidoc

```diff
@@ -1,4 +1,111 @@
 [[logs-troubleshooting]]
 = Troubleshoot your logs
 
-_coming soon_
\ No newline at end of file
+This section provides solutions to errors you might encounter while onboarding your logs in {kib}.
+
+[discrete]
+[[logs-troubleshooting-insufficient-priv]]
+== User does not have permissions to create API key
+
+If you don't have the required privileges to create an API key, you'll see the following error message:
+
+----
+User does not have permissions to create API key.
+
+Required cluster privileges are [`monitor`, `manage_own_api_key`] and 
+required index privileges are [`auto_configure`, `create_doc`] for 
+indices [`logs-*-*`, `metrics-*-*`], please add all required privileges 
+to the role of the authenticated user.
+----
+
+*Solution:*
+
+You need an administrator to give you the necessary permissions listed in the error messages and then to restart the onboarding flow.
+
+[discrete]
+[[logs-troubleshooting-API-key-failed]]
+== Failed to create API key
+
+If you don't have the privileges to create `savedObjects` in {kib}, you'll see the following error message:
+
+----
+Failed to create API key
+
+Something went wrong: Unable to create observability-onboarding-state
+----
+
+*Solution:*
+
+You need an administrator to give you privileges to create `savedObjects` in {kib} to generate the required `observabilityOnboarding` flow state.
```

Comment:
To make this more actionable, could we share here the API call that the admin would have to run?

Reply:
@ruflin Could someone provide this API call? Not sure exactly how to format that.

Reply:
@weltenwort Could someone in your team help out here?

Reply:
Are we looking for an API call to grant the privileges? Or instructions to assign them via Kibana's management UI?

Reply:
Hm, that's a bit tricky as much of the call depends on the specific user and its roles. They'd need to add these privileges to one of the roles of the current user, but we don't know how they've structured their roles. And if the user doesn't have these privileges, they probably don't have the privilege to grant new privileges either.

Would it maybe make sense to ship with a default observability role that grants the privileges and instruct them to ask the admin to grant that role?

Reply:
@ruflin any suggestion here? How do we want to move forward with the docs?

Reply:
Lets take this on in steps. Lets first document the permissions that are needed. From there, we can discuss if we should have a role for it and API calls.